nncf/common/logging/track_progress.py

```diff
@@ -105,13 +176,14 @@ def __init__(
                     complete_style=complete_style,
                     finished_style=finished_style,
                     pulse_style=pulse_style,
+                    bar_width=None,
                 ),
                 TaskProgressColumn(show_speed=show_speed),
```

Comment:
I would personally like to keep percent because it's a clear indication on the fraction of progress, otherwise one would need to estimate division of two numbers